package.json

```diff
@@ -48,6 +48,8 @@
     "linkify-it": "^2.0.3",
     "lodash": "^4.6.1",
     "mobile-detect": "^1.2.1",
+    "react-addons-shallow-compare": "^15.6.0",
```

Comment:
Unfortunately yes, it's needed by [react-dates](https://github.com/airbnb/react-dates) (see `Install dependencies` section)